source/sass/blocks/header.scss

```
@@ -40,11 +50,123 @@
 
 .header__logo {
   display: block;
-  padding: 20px;
+
+  @media (max-width: $mobile-width-only) {
+    .header__logo--tablet {
+      display: none;
+    }
+
+    .header__logo--desktop {
+      display: none;
+    }
+  }
+
+  @media (min-width: $tablet-width) {
+    .header__logo--mobile {
+      display: none;
+    }
+
+    .header__logo--desktop {
+      display: none;
+    }
+  }
+
+  @media (min-width: $desktop-width) {
+    .header__logo--desktop {
+      display: block;
+    }
+
+    .header__logo--molile {
+      display: none;
+    }
+
+    .header__logo--tablet {
+      display: none;
+    }
+  }
+}
+
+.header__logo:hover {
+  svg path {
+    fill: $color-accent;
+  }
+}
```

Тройная вложенность получается и тут тоже.
Кстати, интересно, почему именно так сформирована вложенность...
А например не так )
```suggestion
.header__logo:hover svg path {
  fill: $color-accent;
}
```

Или так!
```suggestion
.header__logo {
  &:hover svg path {
    fill: $color-accent;
  }
}
```

Но это уже так, баловство. Главное конечно количество уровней вложенности